include/quicly/frame.h

```diff
@@ -62,6 +62,7 @@ extern "C" {
 #define QUICLY_FRAME_TYPE_STREAM_BIT_LEN 0x2
 #define QUICLY_FRAME_TYPE_STREAM_BIT_FIN 0x1
 
+#define QUICLY_MAX_ACK_FRAME_RANGE 256
```

Comment:
Adding a definition is a good idea :+1:

OTOH, this should be `QUICLY_MAX_ACK_RANGE_COUNT` or something. "Range" of an ACK refers to the each block of contiguous packet numbers that are being acked. Not the number of blocks (please refer to https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#rfc.section.19.3.1).